tests/readme_examples.cpp

```diff
@@ -305,6 +305,30 @@ bool is_correct_string() {
   return is_ok;
 }
 
+void parse_documentation() {
+  const char *json      = "{\"key\":\"value\"}";
+  const size_t json_len = strlen(json);
+  simdjson::dom::parser parser;
+  // We comment out element since it is unused.
+  //simdjson::dom::element element = 
+  parser.parse(json, json_len);
+}
+
+
+void parse_documentation_lowlevel() {
+  // such low-level code is not recommended
+  // Motivation: https://github.com/simdjson/simdjson/issues/1175
```

Comment:
I think this is a bit overreaching. Would it be better to say:

// This shows how to manually handling padding.
// The recommendation is to only use this if the extra complexity of padding
// outweighs the cost of avoiding the internal copy when realloc_if_needed is false.
// See https://github.com/simdjson/simdjson/issues/1175 for reference.

Reply:
Well. This part is for our own purposes, right? So it is not for the end users.

Even so, if a end user does end up browsing this code, I do not want the end user to think "oh! that's how I should use this API". It is a "last resort" code example for people who cannot do better.

I will tone down the language, but I do want users to be warned off.